screwdriver.yaml

```diff
@@ -0,0 +1,28 @@
+shared:
+  image: golang
```

Comment:
I think it might be better to specify the golang image tag.

Reply:
Other Screwdriver project does not set image tag for golang image. I think Screwdriver team are expecting that golang project work properly on new golang docker image. Thus I did not set tag.

Reply:
🙆